run_service.sh

```diff
@@ -738,6 +753,11 @@ echo "-----------------------------------------"
 echo "Checking Autonolas Protocol service state"
 echo "-----------------------------------------"
 
+# We set by default AGENT_ID=14, and check if we are in the Everest
+# staking program before on-chain actions to correct AGENT_ID=12 if 
+# necessary.
+fix_agent_id $USE_STAKING $AGENT_ID $CUSTOM_STAKING_ADDRESS "../$env_file_path"
```

Comment:
I do not think that this is the only thing that should change if we are in Everest, the staking contracts and the params are different there

Reply:
Can you clarify? This line is just fixing the AGENT_ID for everest, which must be 12. For Alpine, it's not modified (it's left as 14)

The staking contracts and params are set up above.

Reply:
- These two parameters https://github.com/valory-xyz/trader-quickstart/pull/196/files#diff-91c4da23ec8a4f27b22bcaaf93a53e4a21cfeeb6a6977e9a3fdd994701ba3b40R577-R578 are true for Everest and not For Alpine
- While this contract address is only correct for Alpine https://github.com/valory-xyz/trader-quickstart/pull/196/files#diff-91c4da23ec8a4f27b22bcaaf93a53e4a21cfeeb6a6977e9a3fdd994701ba3b40R589

Reply:
These have been fixed. We cannot have the same release of quickstart for Everest and Alpine, it will overcomplicate the logic too much. A sensible approach is that
- Until release X, only Everest is supported.
- For any release >X only Alpine is supported.

Reply:
UPDATE: The script requires unstaking from Everest first.
Since AGENT_ID will be always 14 from now on (on new stakings), I am deleting this fix.